### PR TITLE
starter template and move category to top

### PR DESCRIPTION
**Pull request by pbrocks**
Move Categories to top
image

Add starter template
image

### CODE REVIEW

**Comment by mauteri** [comment]
@pbrocks this is similar to https://github.com/GatherPress/gatherpress/issues/76 but the idea was not to have a starter template block, but to set `template` in the event post type when it is registered: https://developer.wordpress.org/block-editor/reference-guides/block-api/block-templates/ This saves a click from setting the default block layout, by presenting an event with the most common blocks to get started. We don't have all the blocks we need yet, so this hasn't been done, but if you want to get started on this that'd be great.

File: core/classes/class-setup.php [hunk 1]
```diff
@@ -147,6 +144,11 @@ public function register_post_types() {
 					'revisions',
 				),
 				'menu_icon'     => 'dashicons-nametag',
+				'template'  => array(
+					array(
+						'gatherpress/event-starter',
```
Comment (mauteri) [inline core/classes/class-setup.php:149]: Okay, going back to my comment before I see you are adding it here. Why would we make this a block too in `event-starter`? I'd rather us not clutter up blocks UI and just call the blocks in template.

**Comment by pbrocks** [comment]
> this is similar to #76 but the idea was not to have a starter template block, but to set `template` in the event post type when it is registered:

That is what I did. I registered the template in the post type, but they can be considered placeholders.

**Comment by mauteri** [comment]
> > this is similar to #76 but the idea was not to have a starter template block, but to set `template` in the event post type when it is registered:
> 
> That is what I did. I registered the template in the post type, but they can be considered placeholders.

Not sure we need this listed in with blocks, I think it just adds clutter. I would just keep it simple and set the choice blocks to the template. I'm thinking within settings at some point we can provide a few layout options and control it from there.

**Pull request closed without merging**